libtiledbsoma/src/soma/managed_query.h

```diff
@@ -265,6 +265,15 @@ class ManagedQuery {
      */
     std::shared_ptr<ArrayBuffers> results();
 
+    /**
+     * @brief Get the array.
+     *
+     * @return std::shared_ptr<Array> Array
+     */
+    std::shared_ptr<Array> array() {
+        return array_;
+    }
+
```

Comment:
Just curious:  why do we need this in managed query now?  It was previously hidden way one layer below.